lib/private/Avatar/Avatar.php

```diff
@@ -151,7 +171,7 @@ protected function generateAvatar(string $userDisplayName, int $size, bool $dark
 		}
 		imagefilledrectangle($im, 0, 0, $size, $size, $background);
 
-		$font = __DIR__ . '/../../../core/fonts/NotoSans-Regular.ttf';
+		$font = $this->getFont($userDisplayName);
```

Comment:
I am not sure how common it is to have roman letter in front of the name, but as the CJK fonts are pretty huge we can improve performance and maybe prevent errors here by using the text only which is rendered (so only check the really needed fonts):

```suggestion
		$font = $this->getFont($text);
```

Reply:
Thanks for the suggestion. I've tested it and submitted the changes.